### PR TITLE
fix: docker-compose-full

### PR DESCRIPTION
**Pull request by martinsaporiti**

### CODE REVIEW

**Comment by x1m3** [comment]
I think PR should go to develop branch.

**Review by x1m3 — APPROVED** [review]

File: infrastructure/local/docker-compose-full.yml [hunk 1]
```diff
@@ -61,8 +61,10 @@ services:
       - "8088:80"
     env_file:
       - ../../.env-ui
+      - ../../.env-issuer
```
Comment (barrybecker4) [inline infrastructure/local/docker-compose-full.yml:64]: what about ../ui/.env? Does that need to be loaded too? When I follow the README, this is what I see:
image

Comment (martinsaporiti) [reply infrastructure/local/docker-compose-full.yml:64]: The `./ui/.env` file is needed only when you run the fe as a developer, executing `npm run ...`. It is not needed when you run the issuer node with docker. 
Regarding your problem, which steps are you following to run the issuer node?

Comment (barrybecker4) [reply infrastructure/local/docker-compose-full.yml:64]: Ok. Never mind. I did finally get the issuer node running locally yesterday, but it was difficult to figure our the nginx and ngrok configuration needed to get past the CORS errors.

**Pull request merged**